cm_sdk/models/alert_config.py

```diff
@@ -0,0 +1,230 @@
+from cm_sdk.models import CloudManagerBase, CloudManagerEnum
+
+
+class AlertConfig(CloudManagerBase):
+    """
+    Class and sub-classes for representing MongoDB CloudManager API objects in python.
+    
+    https://docs.cloud.mongodb.com/reference/api/alert-configurations/
+
+    Child objects are represented and nested classes and enums are
+    used to represent sets of contrained values from the APIs
+    perspective.
+
+    In cases where child object need to be de-serialized from JSON
+    automatically, @property and @.setter methods will need to be
+    provided.
+
+    All consructors expect key-word style arguments.
+
+    """
+
+    def __init__(self, id=None, group_id=None, type_name=None,
+                 event_type_name=None, enabled=False,
+                 matchers=None, notifications=None):
+        self.group_id = group_id
+        self.type_name = type_name
+        self.event_type_name = event_type_name
+        self.enabled = enabled
+        self.matchers = matchers if matchers else []
```

Comment:
Do you mean to default on the falsy case here? It seems like you intended to have the behavior of `if matchers is None else []`.